### DISCUSSION TITLE
Synoai and Deepstack on seperate DSM

### DISCUSSION THREAD

**Discussion by frederikgevaert** [djdd87/SynoAI#214] [Q&A]
Hi, 

I have a Synology with docker support (192.168.40.10) and a seperate synology (no docker support) which I only use for surveillance station. (192.168.40.9)
I followed the manual and installed SynoAI and deepstack on 192.168.40.10, but can't get the setup to work.

This is the appsettings.json file
<img width="313" alt="image" src="https://github.com/djdd87/SynoAI/assets/142431449/88381d99-ed94-4c86-a63d-77588fd38d92">

The trigger gets an response 200, and I see in the logging that the SynoAI is notified. But no files appear in the Capture folder
<img width="407" alt="image" src="https://github.com/djdd87/SynoAI/assets/142431449/a43ba8f1-49b3-4622-85e8-45e6ccd395ca">

<img width="598" alt="image" src="https://github.com/djdd87/SynoAI/assets/142431449/424d32df-94b3-42e2-868c-73218f33c441">

Container file & folder mapping looks ok, and deepstack is activated
<img width="530" alt="image" src="https://github.com/djdd87/SynoAI/assets/142431449/13b6f9ab-e6c0-443b-833a-99cb0c1bbc58">

What could be the issue?

**Comment by JoeyJoe6** [comment]
you can change this to obtain more info in the logs 
"Default": "Debug",

Based on the logs it is waiting for a response from deepstack. The response 200 looks to only be from your synology to synoai

Did you add "Set VISION-DETECTION: True" into your synoai docker Environment tab?

**Comment by djdd87** [comment]
It will only write out the original snapshots if `SaveOriginalSnapshot` is true. And it will only save snapshots if it detects something; your logs do not show any valid detections.

Check your volumes mapping if you think it should be saving files.
